cmd/thv/app/config.go

```diff
@@ -56,6 +57,22 @@ Examples:
 	RunE: setRegistryCmdFunc,
 }
 
+var setRegistryAPICmd = &cobra.Command{
```

Comment:
why have an entirely new field for the registry API? We could have reused the existing `set-registry` which already takes a URL. Probably what we should do instead is have a flag that toggles between the toolhive format and the upstream format. wdyt?